File: 2A_1/cpp/suma_liczb.cpp
/*
 * suma_liczb.cpp
 * 
 * Copyright 2020  <>
 * 
 * Program sumuje liczby z wygenerowanego zakresu
 *
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int suma = 0;  // definicja zmiennej
	int liczba = 0;  // zmienna iteracyjna (sterująca)
	
	int start = 0;
	int koniec = 0;
	cout << "Podaj początek zakresu: ";
	cin >> start;
	cout << "Podaj koniec zakresu: ";
	cin >> koniec;
	
	for (liczba = start; liczba < koniec + 1; liczba++) {
		cout << liczba << endl;
		suma = suma + liczba;
	}
	cout << "Suma: " << suma;
	return 0;
}

File: 2A_1/python/suma_liczb.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  suma_liczb.py
#  
#  Program sumuje liczby z wygenerowanego zakresu


def main(args):
    suma = 0
    start = int(input("Podaj początek zakresu: "))
    koniec = int(input("Podaj koniec zakresu: "))
    
    for liczba in range(start, koniec + 1):
        print(liczba)
        suma = suma + liczba
    print("Suma =", suma)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
